README.md

```diff
@@ -118,9 +140,13 @@ This same technique should work for all of the Adafruit M0/M4 boards, as they al
 
 ## Getting code onto the device: JLink
 
-If you have a board with a SWD debug header, such as the [Metro M0][metro_m0], or if you attached the header yourself, you can  use your JLink together with gdb. @wez prefers using the JLinkGDBServer, but you can also use OpenOCD.
+If you have a board with a SWD debug header, such as the [Metro M0][metro_m0], or if you attached the header yourself, you can use your JLink programmer to either flash the device or debug it (together with gdb).
+
+You can use JFlashLiteExe to just flash the .bin file (note that for some boards, such as [Feather M0][feather_m0], you need to specify the program memory starting address offset or you'll just get a nondescript flashing error).
+
+To debug your board you can run JLinkGDBServer instead of JFlashLiteExe. @wez prefers using the JLinkGDBServer, but you can also use OpenOCD. Note that starting a debug session normally triggers a flashing, so you don't need a separate flashing step before this.
```

Comment:
Isn't this only true if the `load` command is in your .gdbinit?

Reply:
Ah, true. I'll edit that.